omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -171,7 +171,11 @@
             # remove windows specific configs
             delete "#{install_dir}/etc/conf.d/winproc.d"
 
-            # Nothing to move on osx, the confs already live in /opt/datadog-agent/etc/
+            # Codesign everything
+            command "find #{install_dir} -type f | grep -E '(\\.so|\\.dylib)' | xargs codesign --force --timestamp --deep -s 'Developer ID Application: Datadog, Inc. (JKFCB4CN7C)'"
```

Comment:
Doesn't codesign need a private key or something? 🤔

Reply:
The signing identity is defined here: `-s 'Developer ID Application: Datadog, Inc. (JKFCB4CN7C)'`, and the private key + certificate corresponding to this identity need to be in the builder's keychain for this to work